internal/pkg/cli/pipeline_delete.go

```diff
@@ -19,14 +19,14 @@ import (
 )
 
 const (
-	pipelineDeleteConfirmPrompt       = "Are you sure you want to delete pipeline %s from project %s?"
-	pipelineDeleteConfirmHelp         = "This will delete the deployment pipeline for the app(s) in the current project workspace."
+	pipelineDeleteConfirmPrompt       = "Are you sure you want to delete pipeline %s from application %s?"
+	pipelineDeleteConfirmHelp         = "This will delete the deployment pipeline for the services in the workspace."
```

Comment:
This prompt sounds weird. I forget - does pipeline delete only work if there's a local pipeline.yml?

Reply:
I believe so, since we were reading the local manifest to get the pipeline name. We can change this now that we have the codepipeline client though.

Reply:
Yeah, I just checked and what @SoManyHs wrote is correct!